docs/src/modules/java-protobuf/pages/running-locally.adoc

```diff
@@ -0,0 +1,266 @@
+= Run a service locally
+:page-supergroup-java-scala: Language
+include::ROOT:partial$include.adoc[]
+
+Running a service locally is helpful to test and debug. The following sections provide commands for starting and stopping a single service locally.
+
+== Prerequisites
+
+In order to run your service locally, you'll need to have the following prerequisites:
+
+* Docker {minimum_docker_version} or higher
+* Access to the `gcr.io/kalix-public` container registry
+
+[TIP]
+====
+The samples provided with the Kalix SDKs all have `docker-compose` files preconfigured to run your service locally. They also contain the configuration to start a local Google Pub/Sub emulator that the Kalix proxy will connect to when you make use of topic-based Eventing in your service.
+====
+
+== Starting your service
+
+To start your service locally, run the following command from the root of your project:
+
+[.tabset]
+Java::
++
+--
+[source, command line]
+----
+mvn kalix:runAll
+----
+--
+
+Scala::
++
+--
+[source, command line]
+----
+sbt runAll
+----
+--
+
+This command will start your Kalix service and a Kalix Proxy using the included `docker-compose.yml` file.
+
+If you prefer, you can also start `docker-compose` directly by running `docker-compose up` in one terminal and in another terminal start your Kalix service with:
+
+[.tabset]
+Java::
++
+--
+[source, command line]
+----
+mvn kalix:run
+----
+--
+
+Scala::
++
+--
+[source, command line]
+----
+sbt run
+----
+--
+
+== Invoking your service
+
+After you start the service it will accept invocations on `localhost:9000`. You can use gRPCurl to invoke your service.
```

Comment:
```suggestion
After you start the service it will accept invocations on `localhost:9000`. You can use [gRPCurl](https://github.com/fullstorydev/grpcurl) to invoke your service.
```

Reply:
Agree, but need asciidoc syntax.

```suggestion
After you start the service it will accept invocations on `localhost:9000`. You can use https://github.com/fullstorydev/grpcurl[gRPCurl] to invoke your service.
```